src/iop/lens.cc

```diff
@@ -2098,8 +2098,14 @@ static void modflags_changed(GtkWidget *widget, gpointer user_data)
 void gui_changed(dt_iop_module_t *self, GtkWidget *w, void *previous)
 {
   dt_iop_lensfun_params_t *p = (dt_iop_lensfun_params_t *)self->params;
+  dt_iop_lensfun_gui_data_t *g = (dt_iop_lensfun_gui_data_t *)self->gui_data;
 
-  if(p->tca_r != 1.0 || p->tca_b != 1.0) p->tca_override = 1;
+  // enable tca override toggle if tca_b or tca_r is changed
+  if(w == g->tca_b || w == g->tca_r)
+  {
+    p->tca_override = 1;
```

Comment:
As this is now a boolean we also want to clean the setting : 1 -> TRUE

Also please double check any other setting in the code as I suppose we have 0 or 1 instead of TRUE or FALSE.

Reply:
I think there aren't any other places where 0 or 1 was used. The latest commit removes this lines.